tests/conftest.py

```diff
@@ -0,0 +1,25 @@
+import ast
+
+import pycodestyle
+import pytest
+
+from flake8_import_order.flake8_linter import Linter
+
+
+def _get_linter(argv, code, filename):
+    parser = pycodestyle.get_parser('', '')
+    Linter.add_options(parser)
+    options, args = parser.parse_args(argv)
+    Linter.parse_options(options)
+    return Linter(
+        ast.parse(
+            code,
+            filename
+        ),
+        str(filename)
+    )
+
+
+@pytest.fixture
+def flake8_linter():
```

Comment:
@ulope I'm not sure why this function needs to be a fixture, why not just call the function?